### DISCUSSION TITLE
Automatic collections based on child folders

### DISCUSSION THREAD

**Discussion by Hiplobbe** [Kareadita/Kavita#3390] [Ideas]
### Idea Description

I have a pretty extensive TTRPG pdf collection, but without meta data like "series" etc. This makes it hard to have automatic collection creation. 

A way to resolve this would be to make collection based on 1st level child folders, e.g. RPG/DnD but all files in that folder into the "DnD"-collection.

### Idea Category

Feature Enhancement

### Duration of Using Kavita

1h 

### Before submitting

- [X] I've already searched for existing ideas before posting.

**Comment by therobbiedavis** [comment]
Kavita is folder structure agnostic. It wasn't built to look at or follow people's folder structures, and this is by design because people's folder structures are as varied as the content that people read. Only under certain circumstances does it even look at the parent folder (primarily when a series can't be parsed from the filename) otherwise it is entirely dependent on filenames.

We've looked into supporting metadata for PDFs but there's not a whole lot of non-propietary support out there for C#, additionally the metadata categories PDFs do use are garbage for this sort of use-case.

I guess that's a long way of saying that I don't think this will be considered. I would suggest converting your PDFs to Images and zipping them up into CBZs and adding metadata.
